packages/docs/page-config/ui-elements/value/examples/WithInputs.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div v-for="data, key in form" class="item">
```

Comment:
```suggestion
  <div v-for="(data, key) in form" :key="key" class="item">
```

Reply:
key is not required for primitive items in v-for in vue3.